### DISCUSSION TITLE
Xilinx Versal Series

### DISCUSSION THREAD

**Discussion by sklckdwn** [fastmachinelearning/hls4ml#620] [General]
Hi everyone,

I saw that hls4ml is not yet supporting Vitis AI and recommends using Vivado HLS version =/<2020.1. Those Vivado versions don't support their newer Versal architecture, so I wanted to know if hls4ml can be used on versal chips just like on the ultrascale chips or if something has to be done/added to do so?

**Comment by jmitrevs** [comment]
Vitis support is in active development, with an early pull request already open. So hopefully soon it will be supported.
